src/plugins/autoRowSize/autoRowSize.js

```diff
@@ -367,8 +386,8 @@ class AutoRowSize extends BasePlugin {
    * Clears cached heights.
    */
   clearCache() {
-    this.heights.length = 0;
-    this.heights[-1] = void 0;
+    this.headerRowsHeights.clear();
+    this.rowHeightsMap.setDefaultValues();
```

Comment:
My fault, this function wasn't marked as `private`. It's done from https://github.com/handsontable/handsontable/commit/4252ba8625c0d8bdf4eee01aceb262968945b60a. Please change from `this.columnWidthsMap.setDefaultValues` to `this.columnWidthsMap.clear`.